lib/poller.test.js

```diff
@@ -0,0 +1,127 @@
+'use strict'
+
+const { expect } = require('chai')
+const sinon = require('sinon')
+
+const Poller = require('./poller')
+
+async function delay () {
+  await new Promise((resolve) => setTimeout(resolve, 500))
+}
+
+describe('lib.poller.Poller', () => {
+  let poller
+  const mockSplitChanges = {
+    splitChanges: [
+      { status: 'bar' },
+      { status: 'baz' }
+    ],
+    since: 1
+  }
+  const mockSegmentChanges = {
+    segmentChanges: [
+      { name: 'zoo' }
```

Comment:
does that mean in `segmentChanges` we won't return the `since` value like `splitsChanges`?

Reply:
right, i don't think we need it. we'll have the `since` value for splits which we can use to get updated splits to then pass to the `getSegments` functions (i have that in a separate pr later)